Description: Drop the glibc and stdcxx checks.
 We're not compiling Firefox to run on a different version.
Author: Chris Coulson <chris.coulson@canonical.com>
Author: Olivier Tilloy <olivier.tilloy@canonical.com>
Forwarded: not-needed

--- a/python/mozbuild/mozbuild/action/check_binary.py
+++ b/python/mozbuild/mozbuild/action/check_binary.py
@@ -296,9 +296,7 @@ def checks(target, binary):
         target = HOST
     checks = []
     if target['MOZ_LIBSTDCXX_VERSION']:
-        checks.append(check_stdcxx)
         checks.append(check_libgcc)
-        checks.append(check_glibc)
 
     # Disabled for local builds because of readelf performance: See bug 1472496
     if not buildconfig.substs.get('DEVELOPER_OPTIONS'):

